includes/classes/Feature/BooleanSearchOperators.php

```diff
@@ -86,10 +86,9 @@ public function integrate_boolean_search_operators( $enabled, $query ) {
 			return false;
 		}
 
-		if ( true === $query->query_vars['ep_integrate'] &&
-			( $this->is_active() || true === $query->query_vars['ep_boolean_operators'] ) ) {
+		if ( $this->is_active() ) {
```

Comment:
Can we keep the `true === $query->query_vars['ep_boolean_operators']` here? I'm not sure if someone is relying on that already.

Reply:
I couldn't find any reference of `ep_boolean_operators` in plugin or EP main plugin. However, I added the line back